scripts/release_scripts/generate_release_info_test.py

```diff
@@ -220,19 +220,19 @@ def test_get_changelog_categories(self):
             requester='', headers='',
             attributes={
                 'title': 'PR1', 'number': 1, 'labels': [
-                    {'name': 'CHANGELOG: Test-changes-1 -- @owner1'},
+                    {'name': 'PR CHANGELOG: Test-changes-1 -- @owner1'},
```

Comment:
Change to `PR: CHANGELOG` to be consistent with the other changelog labels.

Reply:
The `:` should be after `PR CHANGELOG` to provide the project names. We are just changing the CHANGELOG label to PR CHANGELOG to ensure that there is no confusion on whether to add this label for issues or PRs.
\cc @seanlip

Reply:
I think this works; alternatively PR_CHANGELOG works too. As long as we have a consistent convention, I don't mind which one we use.